templates/includes/sidebar.html

```diff
@@ -0,0 +1,9 @@
+<!-- templates/includes/sidebar.html -->
+<div class="sidebar">
+  <ul>
+      <li><a href="#">Home page</a></li>
+      <li><a href="#">Manufacturers</a></li>
+      <li><a href="#">Cars</a></li>
+      <li><a href="#">Drivers</a></li>
+  </ul>
+</div>
```

Comment:
The sidebar is correctly implemented with a list of empty links for 'Home page', 'Manufacturers', 'Cars', and 'Drivers', fulfilling the task requirement to create a sidebar with dummy links.